src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAY_PRICE = 40;
+  const LONG_TERM = 7;
+  const MIN_TERM = 3;
+  const MIN_TERM_DISCOUNT = 20;
+  const LONG_TERM_DISCOUNT = 50;
+  const renCost = DAY_PRICE * days;
+
+  if (days >= MIN_TERM && days < LONG_TERM) {
+    return renCost - MIN_TERM_DISCOUNT;
+  }
+
+  if (days >= 7) {
```

Comment:
Don't use literals in expressions (HARDCODE).

Reply:
do it, thanks)))

Reply:
As you can see, this was done on purpose))